app.py

```diff
@@ -83,4 +82,4 @@ def main_panel():
 run_scheduler_cont()
 
 if __name__ == "__main__":
-    app.run(host="0.0.0.0", port=PORT, debug=False)
+    app.run(threaded=True, port=PORT, debug=True)
```

Comment:
Why remove the `host="0.0.0.0"` kwarg?

Reply:
Had added localhost in the Oauth consent screen. Added `0.0.0.0` in the next commit